.test-infra/metrics/grafana/dashboards/perftests_metrics/Python_FnApiRunner_ubenchmarks.json

```diff
@@ -180,29 +180,35 @@
         "current": {
           "$$hashKey": "object:5347",
```

Comment:
Should this be unique? Noticed it last second.

Reply:
I've renamed the variable, so I think it's fine to reuse it (see the definition of it is further down)